core/server/data/migrations/versions/4.9/01-add-default-product-portal-products.js

```diff
@@ -0,0 +1,49 @@
+const {createTransactionalMigration} = require('../../utils');
+const logging = require('@tryghost/logging');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        const products = await knex
+            .select('id')
+            .from('products');
+
+        if (products.length === 0) {
+            logging.warn(`Skipping adding default product to portal_products, no default product exists`);
+            return;
+        }
+
+        const defaultProduct = products[0];
```

Comment:
Is it guaranteed that there's only 1 product in the db at this point?

Reply:
Ideally yes, we never added support for adding more than 1 product and we are treating the first(and only) product as the default everywhere